Sources/Apodini/Components/Create.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  File.swift
+//  
+//
+//  Created by Felix Desiderato on 27.11.20.
+//
+
+import Foundation
+import Fluent
+import Vapor
+
+public protocol DatabaseInjectionContext {
```

Comment:
I love having clearly separate files if you don't mind. Please split up the files a bit